content/en/apps/tutorials/application-tests.md

```diff
@@ -65,6 +65,10 @@ If your `package.json` file does not already have them, add `cht-conf-test-harne
       `death-form-unit-test: "mocha test/forms/death.spec.js --timeout 20000"`
    Then run:
       `npm run death-form-unit-test`
+   |Command|Description|
+   |---|---|
+   |`npm run test`|Compiles the app settings, converts the forms, then runs the tests (preferred solution).
+   |`npm run unittest`|Only runs the tests.|
```

Comment:
It's important to mention these are recommended conventions on how to arrange tests by splitting into unit tests and integration tests so they can each be run independently. This can be best illustrated by sharing an example app such as https://github.com/medic/cht-pih-malawi-app

Reply:
@eljhkrr Done!